src/components/LatestChanges/index.js

```diff
@@ -0,0 +1,215 @@
+import BackButton from '../Layout/BackButton'
+
+const LatestChangesView = () => {
+  return (
+    <>
+      <BackButton />
+
+      <h1 className="govuk-heading-xl govuk-!-margin-bottom-5 govuk-!-margin-top-5">
+        FAQs - Photo functionality in Repairs Hub
+      </h1>
+
+      <div>
+        <img
+          src="/assets/latest-changes-image.png"
+          alt="User guide: Photo functionality in Repairs Hub"
+          style={{ width: '100%', height: 'auto' }}
+        />
+      </div>
+
+      <div className="govuk-body">
+        <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
+          What’s changing and when?
+        </h2>
+        <p className="govuk-!-margin-bottom-5">
+          We are introducing new functionality into Repairs Hub so you will be
+          able to add and view photos on a work order.
+        </p>
+        <p className="govuk-!-margin-bottom-5">
+          This change will be live in Repairs Hub from Tuesday 6 August 2024.
+        </p>
+        <h2 className="govuk-heading-l govuk-!-margin-bottom-5 govuk-!-margin-top-8">
+          How will it impact how I do my job?
+        </h2>
+        <p className="govuk-!-margin-bottom-5">
+          We are asking <strong>operatives</strong> to add relevant photos of
+          the job when closing a work order:
+        </p>
+        <ul className="govuk-list govuk-list--bullet">
+          <li>
+            This is for all circumstances - whether the repair is complete, if
+            there is further work to be carried out, or if it is a no access
```

Comment:
I'm not a native speaker, so it could be that I'm ignorant and clueless, but to me at least, the `whether ... if ... if` sounds a bit odd. If that's correct leave it as is.

Alternatively it could be:

```html
This is for all circumstances:
- when ... complete,
- when ... carried out,
- or when ... no access
```

Reply:
I didnt write it